staging/konvoyconfig/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: "Adds a ConfigMap to the kubeaddons namespace containing the values provided to this chart"
+name: konvoyconfig
+version: 0.0.2
```

Comment:
where did we leave `0.0.1` :) ?

Reply:
you need to add additional properties to be aligned with other charts. Look at the cert-manager-setup as an example.